ExamGenerator.Web.InputModels/Subject/SubjectEditModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using ExamGenerator.Services.Mapping;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace ExamGenerator.Web.InputModels.Subject
+{
+    public class SubjectEditModel : IMapTo<Data.Models.Subject>, IMapFrom<Data.Models.Subject>
```

Comment:
We can reuse SubjectInputModel

Reply:
I dont think so because SubjectEditModel has a property Id. And in the Update I need an id in order to do the update.

Reply:
Am i right?

Reply:
Ok let be so. It is no problem